Kernel/Arch/riscv64/CurrentTime.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023, Sönke Holz <sholz8530@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Arch/CurrentTime.h>
+
+namespace Kernel {
+
+fptr optional_current_time()
```

Comment:
`fptr` ? Is this a type we use in other places? If so, feel free to let me know and we can refactor to a less C-sounding type in a follow-up

Reply:
This weird typedef is from `CurrentTime.h` and all CurrentTime implementations use it.
I didn't want to touch that in this PR as well.